dev/com.ibm.websphere.appserver.mpMetrics-1.0/com.ibm.websphere.appserver.mpMetrics-1.0.feature

```diff
@@ -8,7 +8,7 @@ IBM-ShortName: mpMetrics-1.0
 Subsystem-Name: MicroProfile Metrics 1.0
 -features=com.ibm.websphere.appserver.org.eclipse.microprofile.metrics-1.0, \
  com.ibm.websphere.appserver.cdi-1.2; ibm.tolerates:=2.0,\
- com.ibm.websphere.appserver.javax.annotation-1.2, \
+ com.ibm.websphere.appserver.javax.annotation-1.2; ibm.tolerates:=1.3, \
```

Comment:
I believe this will require an update to the `microProfile-1.2` feature to add:
`com.ibm.websphere.appserver.javax.annotation-1.2; ibm.tolerates:=1.3`
in order to disambiguate the down-level feature inclusion